Gems/Atom/RHI/Vulkan/Code/atom_rhi_vulkan_private_common_files.cmake

```diff
@@ -7,6 +7,8 @@
 #
 
 set(FILES
+    Include/Atom/RHI.Reflect/VkAllocator.h
```

Comment:
These changes will need to be moved to atom_rhi_vulkan_reflect_common_files

Reply:
done